src/GitHub.Exports/Models/DiffUtilities.cs

```diff
@@ -113,6 +115,69 @@ public static DiffLine Match(IEnumerable<DiffChunk> diff, IList<DiffLine> target
             return null;
         }
 
+        public class LineReader
```

Comment:
Since we ended up having a bunch of benchmarks for this, how about adding a comment here with a link to https://gist.github.com/shana/200e4719d4f571caab9dbf5921fa5276?